### DISCUSSION TITLE
Export Prompt History to File with More Details?

### DISCUSSION THREAD

**Discussion by SolusGod** [carson-katri/dream-textures#162] [Ideas]
Hello everyone, I'm in love with this tool and its potential and I wanted to throw an idea that will improve it tremendously. 

The Pitch - Make Prompt History exportable to Txt File with exact details about the seed, resolution, prompt text, samples, CFG Scale and such. Because the AI is so sensitive that changing one letter can provide a whole new generation of concepts and styles, in-depth detailed export of the Prompts and its options are necessary to help you save or recreate the exact Image style and attributes.

This feature would make the history tab a much more dynamic feature with immense power. Saving the heavenly styles/seeds that you discover in this way will help you create a lair where you can lock away *Smegal Voice* the precious.

**Comment by PyrateGFXProductions** [comment]
I was just thinking about this too. It would be nice to be able to export the history to a text file for later. Cheers!

**Comment by difreo** [comment]
Agree, this is very needed. Also bigger multiline prompt window, and permanent place in n panel :)

**Comment by carson-katri** [comment]
This is available in version 0.0.7. See this guide for more info: https://github.com/carson-katri/dream-textures/blob/main/docs/HISTORY.md#prompt-importexport

**Reply by difreo** [reply]
Wow, great update!
